README.md

```diff
@@ -1 +1,139 @@
-# math_react_app
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [Kanban Board](#kanban)
+  - [Explaining Project Video](#project-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Mathmatic App Using React] <a name="about-project"></a>
+
+> This project is build with React.js. For now I have only created the UI of the application but in coming days i am going to add all the button functionality.
+
+## 🛠 Built With <a name="built-with"></a>
+  <ul>
+    <li><a href="https://www.w3schools.com/html">HTML</a></li>
+    <li><a href="https://www.w3schools.com/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/">JS</a></li>
+    <li><a href="https://webpack.js.org/">webPack</a></li>
+    <li><a href="https://jestjs.io/">jest for testing</a></li>
+    <li><a href="https://www.themealdb.com/api.php">mealDB API</a></li>
+    <li><a href="https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270">Involvement API</a></li>
+  </ul>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[User friendly UI]**
+- **[fully functional Calcuator]**
+- **[fully Responsive]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- GETTING STARTED -->
+
+### Prerequisites
+
+In order to run this project you need:
+- Node.js
+- Browser (any)
+- Working PC
+
+### Setup
+
+- In order to Clone this repository to your PC please follow the below steps:
+ > copy this link [https://github.com/Hidar37/math_react_app]
+ > Open your (cmd) or (git bash) and type [cd desktop] to change the directore and selec the desktop
+ > go to your (cmd) or (git bash) and type this command [git clone <copy the repo link>] the repo will be save to your desktop.
+ > run the [npm install]
+ > run [npm start] to see the project in the browser.
+
+### Usage
+
+To run the project, execute the following command:
+
+> Go to that downloaded folder and open the dist/index.html file and your good to go.
+
```

Comment:
- [ ] Here you could add the command `npm start` as the command one has to use in order to execute the app, kindly do so